src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -826,6 +842,12 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_CFLAGS = (
```

Comment:
Why did the CFLAGS bits move around here and in the other place?

Reply:
I moved them from the target to the project, so they apply to both the framework code and the tests.

Reply:
Thanks!